orchagent/notifications.cpp

```diff
@@ -86,3 +87,4 @@ sai_switch_notification_t switch_notifications
     on_switch_shutdown_request,
     NULL
 };
+*/
```

Comment:
switch notification is now passed using SWITCH ATTRIBUTES.

such as 

SAI_SWITCH_ATTR_FDB_EVENT_NOTIFY